packages/manager/modules/bm-server-components/src/network-interfaces/template.html

```diff
@@ -135,23 +135,23 @@
 
         <oui-action-menu data-compact data-placement="top-end">
             <oui-action-menu-item
-                data-ui-sref="app.dedicated-server.server.interfaces.rename({ interface: $row })"
+                data-ui-sref="{{ $ctrl.statePrefix }}.interfaces.rename({ interface: $row })"
```

Comment:
you can also manage it by relative path (as ola component for example)
```suggestion
                data-ui-sref=".interfaces.rename({ interface: $row })"
```

Reply:
It's applied for all places where i use child state.